src/d3a/d3a_core/simulation.py

```diff
@@ -303,7 +303,6 @@ def _execute_simulation(self, slot_resume, tick_resume, console=None):
 
         config.external_redis_communicator.sub_to_aggregator()
         config.external_redis_communicator.start_communication()
-        self._update_and_send_results()
```

Comment:
LGTM, but please clarify why this line was removed

Reply:
https://github.com/gridsingularity/d3a/pull/1032#discussion_r563853049